engine/language_client_codegen/src/typescript/templates/types.ts.j2

```diff
@@ -1,4 +1,8 @@
-import { Image } from "@boundaryml/baml"
+import type { Image, Audio } from "@boundaryml/baml"
+
+export type RecursivePartialNull<T> = T extends object
```

Comment:
Do we still use/need `RecursivePartialNull`?

Reply:
This is still being exported for people who might have been importing this in their projects. We are not using it anywhere else in the generated code. We could add a `@depreacted` flag. Thoughts?

Reply:
Added one.